Rakefile

```diff
@@ -3,4 +3,8 @@ require 'rubocop/rake_task'
 
 RuboCop::RakeTask.new
 
-task default: %w(rubocop)
+require 'scraper_test'
+ScraperTest::RakeTask.new.install_tasks
+
+task test: 'test:data'
+task default: %w(rubocop test)
```

Comment:
I've opened a PR for this. https://github.com/everypolitician-scrapers/nigeria-national-assembly/pull/18